ledger/catchpointtracker_test.go

```diff
@@ -1959,10 +1956,21 @@ func TestCatchpointFastUpdates(t *testing.T) {
 			defer wg.Done()
 			ml.trackers.committedUpTo(round)
 		}(i)
+		lastRound = i
 	}
 	wg.Wait()
 	ml.trackers.waitAccountsWriting()
 
+	if ml.trackers.getDbRound() <= basics.Round(proto.CatchpointLookback) {
```

Comment:
```suggestion
	for ml.trackers.getDbRound() <= basics.Round(proto.CatchpointLookback) {
```
lets try few times to commit ... or move `ml.trackers.committedUpTo(lastRound)` into eventually?